From 4970dc092a98e07e2031a4b7eb91b461b8cbe95b Mon Sep 17 00:00:00 2001
From: J. Bruce Fields <bfields@redhat.com>
Date: Mon, 23 Jun 2014 21:51:33 -0400
Subject: [fs] nfsd: Cleanup unneeded including net/ipv6.h

Message-id: <1403560314-32525-152-git-send-email-bfields@redhat.com>
Patchwork-id: 84502
O-Subject: [Patch RHEL7 151/172] NFSD: Cleanup unneeded including net/ipv6.h
Bugzilla: 1111719
RH-Acked-by: Steve Dickson <SteveD@redhat.com>

From: Kinglong Mee <kinglongmee@gmail.com>

Commit 49b28684fdba ("nfsd: Remove deprecated nfsctl system call and
related code") removed the only use of ipv6_addr_set_v4mapped(), so
net/ipv6.h is unneeded now.

Signed-off-by: Kinglong Mee <kinglongmee@gmail.com>
Signed-off-by: J. Bruce Fields <bfields@redhat.com>
Upstream: a30ae94c0797f9de47626eecc43359989447d7a3
Bugzilla: 1111719
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/fs/nfsd/export.c b/fs/nfsd/export.c
index 90d37b6..1d6d7bd 100644
--- a/fs/nfsd/export.c
+++ b/fs/nfsd/export.c
@@ -17,8 +17,6 @@
 #include <linux/exportfs.h>
 #include <linux/sunrpc/svc_xprt.h>
 
-#include <net/ipv6.h>
-
 #include "nfsd.h"
 #include "nfsfh.h"
 #include "netns.h"
-- 
1.7.1